s2/s2-vendor.mk

```diff
@@ -15,6 +15,9 @@
 # This file is generated by device/leeco/s2/setup-makefiles.sh
 
 PRODUCT_COPY_FILES += \
+    vendor/leeco/s2/proprietary/vendor/lib/libaptX-1.0.0-rel-Android21-ARMv7A.so:system/vendor/lib/libaptX-1.0.0-rel-Android21-ARMv7A.so \
```

Comment:
Wrong ordering. You clearly haven't generated these makefiles, you've modified them by hand